pkg/storage/replica_command.go

```diff
@@ -3368,9 +3368,7 @@ func (r *Replica) sendSnapshot(
 	}
 	if err := r.store.cfg.Transport.SendSnapshot(
 		ctx, r.store.allocator.storePool, req, snap, r.store.Engine().NewBatch, sent); err != nil {
-		return &preemptiveSnapshotError{
-			errors.Wrapf(err, "%s: change replicas aborted due to failed preemptive snapshot", r),
```

Comment:
It's redundant; the current range is already getting picked up from the context at the point this error gets logged.